### DISCUSSION TITLE
协作的效率和可能性 - 关于原则的思考

### DISCUSSION THREAD

**Discussion by askender** [qihuju/qihuju.github.io#40] [Ideas]
本文采用类似《天问》的提问体，不过度展开，这种形式信息密度较高。

为什么上都计划会失败？为什么web2的互联网走向不开放？   ~  经济模型？web3的可用工具？
我们做事情是否以商业思维（公司）、娱乐文学艺术哲学思维（大众娱乐和游戏等，奇忽局也是这个思维为主）、科学思维（学术论文arxiv体系）、工程师思维（github、开源）为主？   技术哲学思维（能结合以上几种）呢？   ~   一起做新的方向？
技术哲学思维（比如dweb）未来的发展方向是什么？现在的进展的困境是什么？     ~ 当前来看是大家对新的东西需求不够迫切，会导致这个转变周期特别漫长？
技术哲学的落地欠缺什么？靠什么解决？ ~  经济模型的激励足够合理；有趣的界面；有深度的内容和创作者加入进来协作；清晰的指导思想；科学的构建规划；高自由度的游戏化的协作体验；协议层的开放生态；人类整体经济基础的提升；人类心智的整体提升 ？

集体协作的重要性？ ~ 或者创作者还是个人英雄主义思维？ 
当前人类除了基础科学的突破，经济政治问题之外，最大的困境是什么？ ~ 对于这一点每个人差异极大，似乎没看到这方面的调研统计分析，   芸芸众生，一生所谓何求？
作为人类未来可能的进化方向，除了广义的计算机方面的技术进展，我们还可以依靠的有哪些？
人类交流的效率和质量问题如何突破？ 语言本身的缺憾性和歧义性如何改进？  ~ 媒介的近未来是怎样的？
印刷术与焚书，互联网与屏蔽，似乎后者的媒介更容易被阻塞和控制？  ~   互联网的底层去中心化程度如何提高？切断光纤？开源p2p硬件&天基互联网
假定dweb互联网的基础已经实现（百年的漫长过程？）， 关于协作的应用层的突破是哪些？  ~ 信息的高效双向互联(更好的pingback)； 更好的可视化和互动体验； 更好的版权管理和许可协议和体验（更简易和宽松的版权协议，nft等进化之后的形态）；信息的长期存储保障和增量更新以及信息组织（ipfs/ipns/ipld等）；更好的版本管理和协作体验（better git）; 

dweb上内容方面的应用层优秀产品有哪些？ 可能的形态有哪些?   ~ wiki? content-community? search-engine&knowledge-graph? better-rss-reader? vr&脑机接口？  亦或是没有被构建出来的形态？

我在做一个web3的内容检索系统，协议选用CC，存储用ipfs，赞赏暂时用like，DID暂时用github，技术栈用jamstack, 但问题是内容从何而来？  ~ 已有很多开放数据集；ipfs上有很多文章；潜在的独立博客作者？   问题是创作者喜欢用什么工具？目前来看，独立文本创作者在变少，语音、影像和VR游戏创作者变多。    通过网络，链接文档、版本比较和非顺序写作将创造出一种能够存储和代表人类艺术和科学遗产的“文献宇宙”（docuverse）？

参考：
没有真正的“学科”，只有知识整体。世界上所有的主题之间有着千丝万缕的联系，它们不能被彻底分开。——泰德·尼尔森
程序员有了能用的代码，但没有机器。后来，他们会有机器，但没有能用的代码。

后记：
本文写的比较乱，记录一些零散的思考细节和设计逻辑。
等开发告一段落有了一个公开可用的demo后，再重写本文。

**Comment by Lupingyue** [comment]
很赞！信息量很大，很有高度和前瞻性，需要消化一下。同时期待你开发的demo～
